src/screens/Booking/Booking.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
+
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import {RouteProp, useRoute} from '@react-navigation/native';
+import {FlightData} from '@components';
+import Baggage from './components/Baggage/Baggage';
+import Form from './components/PassengerForm/Form';
+import styles from './Booking.styles';
+import {RootStackParamsList} from '@navigation/types';
+
+const Booking = () => {
+  const route = useRoute<RouteProp<RootStackParamsList, 'BookingScreen'>>();
+  const [state, setState] = React.useState({
+    cabin: true,
+    checked: true,
+  });
+
+  const {
+    id,
+    destination,
+    destinationIata,
+    origin,
+    originIata,
+    time,
+  } = route.params.modalData;
+
+  const setCabinVal = () => setState({...state, cabin: !state.cabin});
+  const setCheckedVal = () => setState({...state, checked: !state.checked});
+
+  return (
+    <View style={styles.container}>
+      <ScrollView style={styles.scrollview}>
+        <FlightData
```

Comment:
price before discount and price after discount is not included, it should be added as specified in the task description